artifactory/utils/commandsummary/buildinfosummary_test.go

```diff
@@ -1,15 +1,16 @@
 package commandsummary
 
 import (
-	buildInfo "github.com/jfrog/build-info-go/entities"
-	buildinfo "github.com/jfrog/build-info-go/entities"
-	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
-	"github.com/stretchr/testify/assert"
 	"os"
 	"path/filepath"
 	"regexp"
 	"strings"
 	"testing"
+
+	buildInfo "github.com/jfrog/build-info-go/entities"
+	buildinfo "github.com/jfrog/build-info-go/entities"
```

Comment:
package "github.com/jfrog/build-info-go/entities" is being imported more than once